AnnoDesigner/Localization/Localization.cs

```diff
@@ -271,6 +272,7 @@ public static void Init(ICommons commons)
                         { "OK" , "OK" },
                         { "Cancel" , "Abbrechen" },
                         { "SelectAll" , "Alle auswählen" },
+                        { "RecentFiles" , "Zuletzt geöffnete Dateien" }
```

Comment:
This should be ""Zuletzt geöffnete dateien" to match the rest of the UI.

Reply:
But in german it is a "Datei" or multiple "Dateien". (I'm a native)